src/Logger/Logger.cc

```diff
@@ -50,31 +103,46 @@ void InitLogger(bool no_log_file, int verbosity) {
             stdout_logger->set_level(spdlog::level::warn);
             break;
         case 5:
```

Comment:
Has 4 (info) disappeared?

In the discussion above we realised that the cleanest way to do this is with separate flags both for performance and for protocol logging (please see my comments below).